.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Create a bug report to help us improve
+title: "[Descriptive title] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **NOTE: General questions should go to the [Discord chat](https://discord.gg/aMxzVcr) instead of the issue tracker.**
+  - type: textarea
+    id: describe
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is and steps to reproduce it.
+      placeholder: This is what I'm seeing.
+    validations:
+      required: true   
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+      placeholder: This is what should happen.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      description: List the steps to reproduce the behavior
+      placeholder: |
+        1. Go to '...'
```

Comment:
Just noticed, maybe we want to make those placeholders a bit more specific to Lodestar to make it easier for users.

Reply:
Do you have an example of what you mean by more specific to Lodestar? The point of keeping it very generic is just to offer a very simple example of how someone should describe listing in order the steps it takes to recreate a bug.

Reply:
Maybe something like

1. Run command '...'
2. Wait node syncing log
3. Invoke API '...'
4. See error

A bit more tailored towards what we expect on how to reproduce an issue on Lodestar and which details we think would be helpful to provide here to diagnose it.